plugins/available/base.plugin.bash

```diff
@@ -91,13 +91,14 @@ function pmdown ()
 
 function mkcd ()
 {
-    about 'make a directory and cd into it'
-    param 'path to create'
+    about 'make one or more directorys and cd into the lastest one'
+    param 'one or more directorys to create'
     example '$ mkcd foo'
     example '$ mkcd /tmp/img/photos/large'
+	example '$ mkcd foo foo1 foo2 fooN'
```

Comment:
Code indentation looks wrong for the two `example` lines, please use the same indentation as the other lines.